### ISSUE TITLE
Bug - Missing alt text on 5_Microservice_Security.md

### ISSUE THREAD

**Issue by nschonni** [canada-ca/platform-security_securite-de-plateforme#30]
### Summary

Several images are missing alt text

### What is the current *bug* behavior?

There is no descriptive text to meet accessibility requirements

### Relevant logs and/or screenshots

```
en/5_Microservice_Security.md:37 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:142 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:244 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:294 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:345 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:417 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:929 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:945 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:978 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:1004 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:1069 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:1155 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:1233 MD045/no-alt-text Images should have alternate text (alt text)
en/5_Microservice_Security.md:1268 MD045/no-alt-text Images should have alternate text (alt text)
```
### Possible fixes

Add relevant descriptions, and re-enable MD045 rule
